src/linear_algebra/matrix.lean

```diff
@@ -421,8 +422,17 @@ def is_basis.det : multilinear_map R (λ i : ι, M) R :=
     intros u i c x,
     simp only [he.to_matrix_update, algebra.id.smul_eq_mul, map_smul_of_tower],
     apply det_update_column_smul
+  end,
+  map_eq_zero_of_eq' := begin
+    intros v i j h hij,
+    rw [←function.update_eq_self i v, h, ←det_transpose, he.to_matrix_update, ←update_row_transpose],
+    have : (he.to_matrix v)ᵀ j = he.repr (v j) := funext (λ _, rfl),
```

Comment:
[Zulip thread](https://leanprover.zulipchat.com/#narrow/stream/217875-Is-there.20code.20for.20X.3F/topic/.60%28he.2Eto_matrix.20v%29.E1.B5.80.20i.20.3D.20he.2Erepr.20%28v.20i%29.60/near/218428932) about whether this lemma should have a name.

Reply:
Seeing in this context, I believe it should have a name, but probably not be a `simp` lemma.